internal/gossiper/metrics.go

```diff
@@ -8,35 +8,65 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const namespace = "gossiper"
+
 type metrics struct {
-	txsReceived     prometheus.Counter
-	seenTxsReceived prometheus.Counter
-	txsGossiped     prometheus.Counter
+	txsReceived            prometheus.Counter
+	seenTxsReceived        prometheus.Counter
+	txsGossiped            prometheus.Counter
+	selectTxsToGossipCount prometheus.Counter
+	selectTxsToGossipSum   prometheus.Gauge
+	selectedTxsToGossip    prometheus.Counter
+	targetTxsSum           prometheus.Gauge
```

Comment:
Is there a specific reason to use a Gauge? I feel like only Inc/Add method are used to represent the metric.